sample.env

```diff
@@ -36,3 +36,40 @@ KAFKA_BROKER_ADDRESS_1=localhost:9092
 # Used to sign transactions and stop the protocol.
 # Make sure there are enough ETH on the balance to send a transaction to stop the protocol
 WALLET_PRIVATE_KEY=0x0000000000000000000000000000000000000000000000000000000000000000
+
+# KEYS API
+
+# App
+KEYS_API_PORT=3001
+
+# CORS
+KEYS_API_CORS_WHITELIST_REGEXP=^https?://(?:.+?\.)?(?:lido|testnet|mainnet)\.fi$
```

Comment:
keys api should not be externally available here. let's leave it blank to avoid unnecessary questions